.github/workflows/ci.yml

```diff
@@ -47,9 +47,11 @@ env:
   PERCY_TOKEN_PIE_TEXTAREA: ${{ secrets.PERCY_TOKEN_PIE_TEXTAREA }}
   PERCY_TOKEN_PIE_TOAST: ${{ secrets.PERCY_TOKEN_PIE_TOAST }}
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  PERCY_DEBUG: "*"
+  LOG_LEVEL: debug
```

Comment:
These are for debugging and I need to keep these in for the moment whilst I keep an eye on people's builds and see if they fail after this fix. Will create another PR to remove after I'm confident the fixes added have resolved our issues.